app/assets/stylesheets/partials/_contact-form.scss

```diff
@@ -0,0 +1,59 @@
+form#contact-form {
+  width: 100%;
+
+  label {
+    color: #666;
+    font-size: 15px;
+    font-weight: bold;
+    margin-bottom: 8px;
+  }
+
+  textarea, input[type="text"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own line